pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -27,14 +26,26 @@ const resolvConf = "/etc/resolv.conf"
 const defaultDNSPort = 53
 
 var errInvalidNetworkUpstream = fmt.Errorf("The address field is mandatory for upstream of type Network, but was not provided")
+var errTransportTLSConfiguredWithoutServerName = fmt.Errorf("The ServerName field is mandatory when configuring TLS as the DNS Transport")
+var errTransportTLSConfiguredForNonIP = fmt.Errorf("Configuring TLS as the DNS Transport is only allowed for IP addresses")
```

Comment:
I think this wording is a little awkward.

What about:
`Only IP addresses are allowed when configuring TLS as the DNS Transport`

Reply:
I was not sure about it either. I liked your suggestion!